libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -619,7 +624,14 @@ void AP_Mount_Backend::set_rctargeting_on_rcinput_change()
     const int16_t yaw_in = (yaw_ch == nullptr) ? 0 : yaw_ch->get_radio_in();
 
     // if not in RC_TARGETING or RETRACT modes then check for RC change
-    if (get_mode() != MAV_MOUNT_MODE_RC_TARGETING && get_mode() != MAV_MOUNT_MODE_RETRACT) {
+    if (last_rc_input.roll_in == 0 && last_rc_input.pitch_in == 0 && last_rc_input.yaw_in == 0)
+    {
+        // The first time through, initial RC  inputs should not reset the mode
+        last_rc_input.roll_in = roll_in;
+        last_rc_input.pitch_in = pitch_in;
+        last_rc_input.yaw_in = yaw_in;
+    }
+    else if (get_mode() != MAV_MOUNT_MODE_RC_TARGETING && get_mode() != MAV_MOUNT_MODE_RETRACT) {
```

Comment:
we can remove this else.  It should be fine for it to run through this check the first time it gets RC input.

Reply:
I've restricted the code to make it clearer.